packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -75,14 +86,28 @@ export default class QuanticSort extends LightningElement {
   schemas = {};
 
   labels = {
-    sortBy,
     relevancy,
     newest,
     oldest,
     invalidCustomSortConfig,
+    sortBy,
+  };
+
+  variants = {
+    default: {
+      name: SORT_VARIANTS.DEFAULT,
+    },
+    wide: {
+      name: SORT_VARIANTS.WIDE,
+    },
   };
 
   connectedCallback() {
+    if (!Object.values(SORT_VARIANTS).includes(this.variant)) {
+      console.warn(
+        `Unsupported variant: ${this.variant} specified in the QuanticSort component, defaulting to the ${SORT_VARIANTS.DEFAULT} variant.`
```

Comment:
This sentence will read:

Unsupported variant: ABC specified in the QuanticSort component, defaulting to the default variant.

I would change it to :


```suggestion
        `Unsupported variant: ${this.variant} specified in the QuanticSort component, using the default variant.`
```

Reply:
sure I mean I thought it was clear enough, but yeah I can change it